Wancer/Wancer/Models/Group.swift

```diff
@@ -30,16 +29,16 @@ class Group: Identifiable {
     }
     
     func addUser(_ user: User) {
-        users.append(user)
+        users?.append(user)
```

Comment:
Can you handle if users is nil? Appending on nil would do nothing. Try appending if users exist and then setting if users does not